src/main/java/com/google/sps/servlets/AddAvailabilityServlet.java

```diff
@@ -60,7 +47,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String tutorID = request.getParameter("tutorEmail");
+        //Set default value to -1 
+        String tutorID = Optional.ofNullable(request.getParameter("tutorID")).orElse("-1");
```

Comment:
Is there ever a case in which the id would end up being -1? Isn't it handles by the datastore itself?

Reply:
I think Datastore only assigns positive IDs. They advise against using negative IDs in datastore (https://cloud.google.com/datastore/docs/best-practices#keys), so I don't think Datastore would ever generate a negative ID. In this case, if the tutorID is null, it will be assigned -1 for the ID. I'll add a check to make sure that if the ID is -1, it should not create a new availability.

Reply:
Cool! It might be a good idea to add that check to the other places where the id is being set to -1 too

Reply:
Yep, just made a commit for these checks.